frontend/public/components/persistent-volume-claim.jsx

```diff
@@ -235,7 +302,41 @@ export const PersistentVolumeClaimsList = (props) => (
     virtualize
   />
 );
+
+const FETCH_METRICS = window.innerWidth >= 1450;
+
 export const PersistentVolumeClaimsPage = (props) => {
+  const { namespace = undefined } = props;
+  const query = getQuery();
+  const dispatch = useDispatch();
+  let pvcMetrics = [];
+  if (FETCH_METRICS) {
+    // Disabling because this would not change between rerenders as screen size change would not retrigger this.
+    // eslint-disable-next-line react-hooks/rules-of-hooks
```

Comment:
IIUC hooks should not be conditional because the condition might change during re-renders but the value that we get for FETCH_METRICS is derived from screen width which does not change since we are listening to the screen size change event. We could instead use fetch here and avoid disabling this rule but it's basically the same thing.

Reply:
I still think in order to future proof this and make it consistent with the rest of our code, this should still follow the rules of hooks.

Reply:
Yes, it's true that the rule is there to ensure hooks always execute on every render. Even so, I think for consistency and future safety, we should still follow the rule here.

Reply:
Removing the whole guard for screen size.